docs/crystal_for_rubyists/README.md

```diff
@@ -105,6 +105,14 @@ will generate an error message like
     in line 1: too many block arguments (given 2, expected maximum 1)
 ```
 
+Following code works.
+
+```cr
+[[1, "A"], [2, "B"]].each do |e|
+  pp e[0]
+  pp e[1]
+end
+```
```

Comment:
I'd recommend the explicit unpacking syntax which is easier to use than manual deconstructuring:

````suggestion
[Block parameter unpacking](https://crystal-lang.org/reference/syntax_and_semantics/blocks_and_procs.html#unpacking-block-parameters) works by wrapping the parameters in parenthesis:

```cr
[[1, "A"], [2, "B"]].each do |(a, b)|
  pp a
  pp b
end
```
````

Reply:
The example you added here was exists in this document, at a position further back from the current position.

I add the `do |e|` version just for the completeness, that is, list all work case.

Reply:
I don't this is good. Putting this on front encourages to use this variant, when a more concise option exists.

Reply:
I moving my example below the original `do |(a, b)|` one, i consider let user know all possible usage is better.